src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -320,7 +322,7 @@ private void CloseExtensionTab(TabItem tabitem)
             // get the selected tab
             TabItem selectedTab = tabDynamic.SelectedItem as TabItem;
 
-            if (tabToBeRemoved != null)
+            if (tabToBeRemoved != null && ExtensionTabItems.Count > 0)
```

Comment:
Is there a reason for adding this condition?

Reply:
When you close the documentation browser, it un-checks the menu item and that will in-turn call the MenuItemUnCheckedHandler() here: https://github.com/DynamoDS/Dynamo/blob/1e77d23ebd980ed42932ecc7e47dcae5234a486e/src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs#L83. So when it tries to close the side bar extension again, it was failing. This would just make sure there are extension tabs present before it closes any.